src/Sarif.FunctionalTests/ConverterTestData/Semmle/results.csv

```diff
@@ -0,0 +1,80 @@
+"TODO/FIXME comments","A comment that contains 'TODO' or 'FIXME' may indicate code that is incomplete or broken, or it may highlight an ambiguity in the software's specification.","recommendation","TODO/FIXME comment.","src/main/java/org/apache/commons/io/FileUtils.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/FileUtils.java","1138","50","1138","64"
+"TODO/FIXME comments","A comment that contains 'TODO' or 'FIXME' may indicate code that is incomplete or broken, or it may highlight an ambiguity in the software's specification.","recommendation","TODO/FIXME comment.","src/main/java/org/apache/commons/io/FileUtils.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/FileUtils.java","1139","51","1139","65"
+"TODO/FIXME comments","A comment that contains 'TODO' or 'FIXME' may indicate code that is incomplete or broken, or it may highlight an ambiguity in the software's specification.","recommendation","TODO/FIXME comment.","src/main/java/org/apache/commons/io/filefilter/MagicNumberFileFilter.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/filefilter/MagicNumberFileFilter.java","264","79","264","114"
+"TODO/FIXME comments","A comment that contains 'TODO' or 'FIXME' may indicate code that is incomplete or broken, or it may highlight an ambiguity in the software's specification.","recommendation","TODO/FIXME comment.","src/main/java/org/apache/commons/io/output/TeeOutputStream.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/output/TeeOutputStream.java","32","38","32","70"
+"Commented-out code","Commented-out code makes the remaining code more difficult to read.","warning","This comment appears to contain commented-out code.","src/main/java/org/apache/commons/io/input/BOMInputStream.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/input/BOMInputStream.java","290","9","293","54"
+"Confusing overloading of methods","Overloaded methods that have the same number of parameters, where each pair of corresponding parameter types is convertible by casting or autoboxing, may be confusing.","recommendation","Method IOUtils.toInputStream(..) could be confused with overloaded method [[""toInputStream""|""relative://src/main/java/org/apache/commons/io/IOUtils.java:1521:59539:13""]], since dispatch depends on static types.","src/main/java/org/apache/commons/io/IOUtils.java","commons-io/revision-2016-November-08--14-46-36/src/main/java/org/apache/commons/io/IOUtils.java","1472","31","1472","43"
```

Comment:
What problem, and what convention? Are you talking about the colon-separated region components 1521:59539:13 in the bracketed link? Not sure what you think the converter should do here. If we do the work to extract these links (I filed a work item for that), then we would parse these out when we created the related location. Although really, to make that safe, they ought to specify a fragment: '.../commons/io/IOUtils.java#1521:59539:13'.

---
In reply to: [90079504](https://github.com/Microsoft/sarif-sdk/pull/545#discussion_r90079504) [](ancestors = 90079504)